lib/block_view_types.go

```diff
@@ -1876,11 +1876,9 @@ func (op *UtxoOperation) RawDecodeWithoutMetadata(blockHeight uint64, rr *bytes.
 			return errors.Wrapf(err, "UtxoOperation.Decode: Problem reading PrevValidatorEntry: ")
 		}
 
-		// PrevGlobalStakeAmountNanos
-		if prevGlobalStakeAmountNanos, err := VariableDecodeUint256(rr); err == nil {
-			op.PrevGlobalStakeAmountNanos = prevGlobalStakeAmountNanos
-		} else {
-			return errors.Wrapf(err, "UtxoOperation.Decode: Problem reading PrevGlobalStakeAmountNanos: ")
+		// PrevGlobalActiveStakeAmountNanos
+		if op.PrevGlobalActiveStakeAmountNanos, err = VariableDecodeUint256(rr); err != nil {
```

Comment:
Nit: No need to fix this here, but I tend to prefer breaking assignments out of if statements like the following. I know it's not "Go-like" but I find it easier to read and less error-prone:

```
		// PrevGlobalActiveStakeAmountNanos
        op.PrevGlobalActiveStakeAmountNanos, err = VariableDecodeUint256(rr)
		if err != nil {
			return errors.Wrapf(err, "UtxoOperation.Decode: Problem reading PrevGlobalActiveStakeAmountNanos: ")
		}
```